README.md

```diff
@@ -232,6 +232,65 @@ values are merged with per section values.
 | `manual` | · | | Required (unlike Cabal) |
 | `default` | · | | Required (unlike Cabal) |
 
+#### <a name="dependencies"></a> Dependencies
+
+Dependencies can be specified as either a list or an object. These are
+equivalent:
+
+```
+  depends:
```

Comment:
Again, this is not a proper review yet, but just that we don't forget latter on, this should be `dependencies` instead of `depends`.

Other than that, the docs look nice :).